config/prow/jobs/config.yaml

```diff
@@ -3428,6 +3428,70 @@ presubmits:
       - name: test-account
         secret:
           secretName: test-account
+  - name: pull-knative-test-infra-check-config
+    agent: kubernetes
+    context: pull-knative-test-infra-check-config
+    always_run: true
+    rerun_command: "/test pull-knative-test-infra-check-config"
+    trigger: "(?m)^/test (all|pull-knative-test-infra-check-config),?(\\s+|$)"
+    decorate: true
+    path_alias: knative.dev/test-infra
+    branches:
+    - "release-0.10"
```

Comment:
This is weird..... we don't have branches in test-infra

Reply:
I know... But for some reason, we'll have to have at least one branch to get the config generator work. I have changed the branch name to `legacy` to be less confusing.
(Config generator is not related to this PR and should have separate discussion.)